examples/text-generation/generation_gemma.py

```diff
@@ -101,14 +107,13 @@ def main():
     pos_ids = pos_ids.max(axis=-1)[0].unsqueeze(1) + 1
 
     model = conditional_compile(model)
-    cache_position = torch.tensor([sequence_length + 1], device=device)
+    cache_position = torch.tensor([sequence_length], device=device)
```

Comment:
What changed to not need the `+1` here?

Reply:
It was actually an error on my side, I figured it out while debugging: I do the +1 further down in the loop just after `decode_one_tokens`, and the right way to initialize it here is to the value of the sequence length.